src/coreclr/src/vm/eventpipe.cpp

```diff
@@ -100,9 +101,18 @@ void EventPipe::Initialize()
         if (tracingInitialized)
             s_state = EventPipeState::Initialized;
     }
-#ifdef FEATURE_EVENTPIPE_STARTUP
```

Comment:
Do we have test coverage on the new code paths now that it is going live?

Reply:
So far I'm using manual tests but I am planning to check in tests into diagnostics repo with the rest of EventPipe tests.